predicators/spot_utils/spot_utils.py

```diff
@@ -255,18 +277,19 @@ def get_camera_images(self) -> Dict[str, Image]:
             camera_images[source_name] = img
         return camera_images
 
-    def get_single_camera_image(self, source_name: str) -> Tuple[Image, Any]:
+    def get_single_camera_image(self,
+                                source_name: str,
+                                to_rgb: bool = False) -> Tuple[Image, Any]:
         """Get a single source camera image and image response."""
-        # Get image and camera transform from source_name.
         img_req = build_image_request(
             source_name,
             quality_percent=100,
             pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8)
         image_response = self.image_client.get_image([img_req])
 
         # Format image before detecting apriltags.
-        if image_response[0].shot.image.pixel_format == image_pb2.Image.\
-            PIXEL_FORMAT_DEPTH_U16:
+        if image_response[0].shot.image.pixel_format == image_pb2.Image. \
```

Comment:
can some of this be factored out to use the new helper function for image response processing?